pkg/printers/internalversion/printers.go

```diff
@@ -1579,10 +1579,10 @@ func printHorizontalPodAutoscaler(obj *autoscaling.HorizontalPodAutoscaler, opti
 	reference := fmt.Sprintf("%s/%s",
 		obj.Spec.ScaleTargetRef.Kind,
 		obj.Spec.ScaleTargetRef.Name)
-	minPods := "<unset>"
+	minPods := int32(1)
```

Comment:
The server is responsible for handling defaulting.  We cannot interpret it on the client.

Reply:
@smarterclayton  thanks, I will close this PR. =)